src/linux_mcp_server/connection/ssh.py

```diff
@@ -242,10 +250,40 @@ async def close_all(self):
 _connection_manager = SSHConnectionManager()
 
 
+def get_bin_path(command: str) -> str:
+    """Get the full path to an executable.
+
+    Raises VauleError if not found.
+    """
+    sbin_paths = ("/sbin", "/usr/sbin", "/usr/local/sbin")
+    paths = os.getenv("PATH", "").split(os.pathsep)
+    for path in sbin_paths:
+        if path not in paths:
+            paths.append(path)
+
+    path = os.pathsep.join(paths)
```

Comment:
Not a necessary change, but for me, it makes the code clear if we don't override the variables with new data in loops like this


```suggestion
    env_paths = os.getenv("PATH", "").split(os.pathsep)
    paths = []
    for path in sbin_paths:
        if path not in env_paths:
            paths.append(path)

    path = os.pathsep.join(paths)
```